crates/mcp-server/src/components.rs

```diff
@@ -60,15 +60,18 @@ pub(crate) async fn handle_load_component(
 
             let contents = vec![Content::text(status_text)];
 
-            if let Some(peer) = server_peer {
-                if let Err(e) = peer.notify_tool_list_changed().await {
-                    error!("Failed to send tool list change notification: {}", e);
-                } else {
-                    info!(
-                        "Sent tool list changed notification after loading component {}",
-                        id
-                    );
-                }
+            info!(
+                "Notifying server peer about tool list change after loading component {}",
```

Comment:
Clippy is now counting non-interpolated variables in format strings as warnings. Also, `id` should be a field on the log and not in the log string (here and in other lines)